examples/bbox_detection/labelme2voc.py

```diff
@@ -111,10 +128,11 @@ def main():
                     maker.truncated(),
                     maker.difficult(),
                     maker.bndbox(
-                        maker.xmin(str(xmin)),
-                        maker.ymin(str(ymin)),
-                        maker.xmax(str(xmax)),
-                        maker.ymax(str(ymax)),
+                        # need to write as int
+                        maker.xmin(str(int(xmin))),
+                        maker.ymin(str(int(ymin))),
+                        maker.xmax(str(int(xmax))),
+                        maker.ymax(str(int(ymax))),
```

Comment:
same here

Reply:
thanks for the reviews!
yes, i reverted this part.

Reply:
used sort to write in a pythonic way :)
```
            # swap if min is larger than max.
            xmin, xmax = sorted([xmin, xmax])
            ymin, ymax = sorted([ymin, ymax])
```